setup_scripts/install_libsbp.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "================================================================"
+echo "Installing Swift Navigation's Binary Protocol library || (GPS)"
+echo "================================================================"
+
+cd external_libs/libsbp/c # get to build space
```

Comment:
```suggestion
cd ../external_libs/libsbp/c # get to build space
```
The path was not relative to `$CURR_DIR` from the top level script.